vcbuild.bat

```diff
@@ -440,6 +440,8 @@ mkdir %TARGET_NAME%\node_modules > nul 2>nul
 
 copy /Y node.exe %TARGET_NAME%\ > nul
 if errorlevel 1 echo Cannot copy node.exe && goto package_error
+copy /Y ..\tools\msvs\node %TARGET_NAME%\ > nul
+if errorlevel 1 echo Cannot copy node && goto package_error
```

Comment:
How does this affect non-WSL? What happens when I type `node` in cmd.exe? In cygwin's bash.exe?